src/web/core/src/main/java/org/geoserver/web/data/layer/CoverageViewEditor.java

```diff
@@ -40,23 +47,34 @@
     List<CoverageBand> currentOutputBands;
     ListMultipleChoice<String> coveragesChoice;
     CompositionType compositionType;
+    IModel<EnvelopeCompositionType> envelopeCompositionType;
+    IModel<SelectedResolution> selectedResolution;
+    IModel<String> resolutionReferenceCoverage;
+
+//    List<EnvelopeCompositionType> envelopeCompositionModes;
+//    private DropDownChoice<EnvelopeCompositionType> envelopeComposition;
```

Comment:
I think we can get rid of these commented pieces since they are created on the fly in the method below